lightning/src/ln/channelmanager.rs

```diff
@@ -1464,6 +1464,8 @@ pub struct ChannelDetails {
 	///
 	/// [`confirmations_required`]: ChannelDetails::confirmations_required
 	pub is_channel_ready: bool,
+	/// Whether the channel is closing and its shutdown stage. `None` for `ChannelDetails` serialized on LDK versions prior to 0.0.116.
```

Comment:
nit: No big deal, but we could just make this:
```suggestion
	/// The stage of the channel's shutdown.
	/// `None` for `ChannelDetails` serialized on LDK versions prior to 0.0.116.
```

For me at least, current wording (other than the `None` explanation that follows) may suggest that "Whether the channel is closing" maps to Some/None, but that's not the case.

Reply:
Ah yeah, I suggested the current wording to clarify that the channel not be shutting down, but after adding the `Option` this prob makes more sense